docs/imagecustomizer/api/configuration/partition.md

```diff
@@ -69,3 +69,29 @@ Supported options:
   This flag is only supported on GPT formatted disks.
 
   For further details, see: https://en.wikipedia.org/wiki/BIOS_boot_partition
+
+- `home`: A `/home` partition.
+
+- `linux-generic`: A generic Linux partition.
+
+  This is the default value.
+
+- `root`: The `/` partition.
+
+- `root-verity`: The verity hash partition for `/`.
+
+- `srv`: The `/srv` partition.
+
+- `swap`: A swap partition.
+
+- `tmp`: The `/var/tmp` partition.
+
+- `usr`: The `/usr` partition.
+
+- `usr-verity`: The verity hash partition for `/usr`.
+
+- `var`: The `/var` partition.
+
+- `xbootldr`: The `/boot` partition.
+
+In addition, a UUID string may be specified.
```

Comment:
Isn't it alternatively? (not in addition)
also either here or in the code, could be good to mention where the uuids are coming from (ie link to the list / table with uuids)

Reply:
I changed the format so that the UUID string option is part of the list.